.github/workflows/openvex.yaml

```
@@ -0,0 +1,35 @@
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        type: string
+        required: true
+        description: "GitHub release tag"
+
+permissions: {}
+
+jobs:
+  vexctl:
+    permissions:
+      contents: write
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Set env
+        run: echo "RELEASE_VERSION=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
+      - name: Set tag name
+        shell: bash
+        run: |
+          TAG=${{ github.event.inputs.tag }}
+          echo "TAG=$TAG" >> "$GITHUB_ENV"
+      - uses: openvex/generate-vex@31b415924ea0d72ed5f2640f1dee59dea6c2770b
+        name: Run vexctl
+        with:
+          product: pkg:generic/karpenter@${{ env.RELEASE_VERSION }}
```

```suggestion
          product: pkg:generic/karpenter@${{ github.ref_name }}
```

I'm assuming that we migth be able to just use the `ref_name` directly here rather than having to parse it out